src/components/cc-product-card/generate-random-keywords.js

```diff
@@ -0,0 +1,10 @@
+export function generateRandomKeywords (number) {
+  const keywords = [{ value: 'hidden', hidden: true }];
+
+  for (let i = 0; i < number; i++) {
+    const str = Math.random().toString(36).substring(2, 7);
```

Comment:
suggestion: Maybe, you could extract this random string generator into `src/lib/utils.js` which already contains some random related functions.